rs/crypto/ed25519/src/lib.rs

```diff
@@ -108,6 +108,23 @@ impl PrivateKey {
         Self { sk }
     }
 
+    /// Generate a test key using a seed and string
+    pub fn generate_test_key(seed: &[u8], key_id: &str) -> Self {
```

Comment:
Same comment as for the corresponding ecdsa function

Reply:
Changed to `generate_from_seed` as discussed in the team meeting